<launch>
    <node name="face_detector" pkg="task_3" type="face_detector.py" output="screen" args="$(find task_3)/data/deploy.prototxt.txt $(find task_3)/data/res10_300x300_ssd_iter_140000.caffemodel" />
    <node name="normal_finder" pkg="task_3" type="normal_finder" output="screen" />

    <node name="move_arm" pkg="task_3" type="move_arm.py" output="log" />
</launch>
